configs/ddppo_pointnav.yaml

```diff
@@ -0,0 +1,62 @@
+BASE_TASK_CONFIG_PATH: "configs/challenge_pointnav2020.local.rgbd.yaml"
+TRAINER_NAME: "ddppo"
+ENV_NAME: "NavRLEnv"
+SIMULATOR_GPU_ID: 0
+TORCH_GPU_ID: 0
+VIDEO_OPTION: []
+TENSORBOARD_DIR: "tb"
+VIDEO_DIR: "video_dir"
+TEST_EPISODE_COUNT: 994
```

Comment:
It this accurate, can we just skip this?

Reply:
This is the exact DDPPO config copied from Habitat-API. Thoughts @erikwijmans ?

Reply:
994 is the number of test episodes in the v1 Gibson pointnav dataset.

Reply:
@erikwijmans anyway we can set it dynamically, or does it have to be hardcoded. :(

Reply:
https://github.com/facebookresearch/habitat-api/pull/379